packages/dom/src/core/plugins/feedback/Feedback.ts

```diff
@@ -52,6 +55,19 @@ export class Feedback extends Plugin<DragDropManager, FeedbackOptions> {
         style.innerText = cssRules;
         document.head.prepend(style);
 
+        // Also inject styles into element document if it differs from host document
+        const element = manager.dragOperation.source?.element;
+
+        if (element) {
+          const elDoc = getDocument(element);
+
+          if (elDoc !== document) {
+            elDocStyle = getDocument(element).createElement('style');
+            elDocStyle.innerText = cssRules;
+            elDoc.head.prepend(elDocStyle);
+          }
+        }
+
```

Comment:
@clauderic this fixes the popover "flashing". dnd-kit needs to handle this rather than user since this needs to be injected before rendering the Feedback plugin.

Reply:
This will prevent the styles from being injected into the host frame - the styles must be in both if dragging between the child and parent frames.

Although my original implementation handles that, your suggestion made me realise that it does _not_ account for the case when dragging between 2 iframes. Perhaps this needs further refactoring to dynamically inject styles into the document for the target element.

Reply:
I've pushed another change based on your suggestion, fixing the above issue: [f7b69d8](https://github.com/clauderic/dnd-kit/pull/1510/commits/f7b69d8df2536b9b7a7a9357342030ee7975dd3e)

There are 2 items outstanding in the PR description (scroller and tidying stories). Let me know if you'd like to do them as part of this, or separately.